joomla/joomla-cms
# Tiny MCE plugin overrides custom settings

Pull Request for Issue # .

### Summary of Changes
Issue reported here: #9812, #8591, #5820, #11851 and #8106
Some of the issues were closed because of the pull requests that came but later it was found out either they do not work or they are wrong(they modify default values).

This change fixes the problem while keeping the default values.

### Testing Instructions
Please test all 3 use cases:

1) Use the following code to display a tinymce editor:
`
$editor = JEditor::getInstance('tinymce');
echo $editor->display('description', "", '250', '50', '60', '5', false);`

2) Add a new article from backend of the site, try to edit this xml code https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_content/models/forms/article.xml#L27 , add width and height attributes (see https://docs.joomla.org/Editor_form_field_type for document about editor field type). For example, change it to;
`<field name="articletext" type="editor"
			label="COM_CONTENT_FIELD_ARTICLETEXT_LABEL" description="COM_CONTENT_FIELD_ARTICLETEXT_DESC"
			filter="JComponentHelper::filterText" buttons="true"  height="50" width="250" />`

3) In the joomla backend go to extensions -> plugins -> tinymce editor -> advanced settings and modify width & height

### Expected result
Tinymce editor has height of 50pixels and width of 250pixels
### Actual result
Tinymce editor has height of 550pixels
### Documentation Changes Required
